dockerfile_skill_basic

```diff
@@ -1,20 +1,19 @@
-FROM ubuntu:latest
+FROM deeppavlov/base:latest
```

Comment:
Does it mean that the latest `deeppavlov` version will be used? We don't need latest, we need the version specifies in `requirements.txt`, otherwise, the code might break.

Reply:
We don't have CUDA-base images for each version of DeepPavlov, I'll try to make Dockerfile, on which relevant version of DeepPavlov can be installed.